README.md

```diff
@@ -1,57 +1,132 @@
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-animate-text
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-animate-text.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-animate-text.svg
+[build-status-url]:https://github.com/moxystudio/react-animate-text/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-animate-text/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-animate-text
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-animate-text/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-animate-text
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-animate-text.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-animate-text?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-animate-text.svg
 
-{package-description}
+A react component that animates text per word and/or per line.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-animate-text
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+With the constant necessity of having animated text creating a component every time becomes time wasting. This component offers a solution to animate text that you can easily integrate and use in your project.
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import AnimateText from '@moxy/react-animate-text';
+
+const MyComponent = (props) => (
+    <div className="container">
+        <AnimateText initialDelay={ 0.5 } wordDelay={ 0.5 }>
+            Lorem ipsum dolor sit amet.
+        </AnimateText>
+    </div>
+);
+
+export default MyComponent;
+```
 
-## API
+The `AnimateText` component uses the [`@moxy/react-split-text`](https://github.com/moxystudio/react-split-text) to split the text.
 
-{package-api-description}
+To import the stylesheet, one can import it on the project's entry CSS file:
 
-#### {package-api-prop-example}
+```css
+@import "@moxy/react-animate-text/dist/index.css";
+```
 
-Type: `object`
-Required: `true`
+Or in the project's entry JavaScript file:
 
-The `{package-api-prop-example}` has the following shape:
 ```js
-{package-api-prop-example}: PropTypes.shape({
-    foo: PropTypes.string,
-    bar: PropTypes.arrayOf(PropTypes.object),
-}).isRequired,
+import "@moxy/react-animate-text/dist/index.css";
+```
+
+## API
+
+These are the props available in `@moxy/react-animate-text`.
+
+#### children
+
+Type: `string` | Required: `true`
+
+Text to be split and animated. The text is split by non-breaking spaces.
```

Comment:
We should take advantage of the `SplitText` component and add the ability to split the text by a [separator](https://github.com/moxystudio/react-split-text#separator) of our choosing.

It should be as simple as adding a `separator` prop and pass it the the `SplitText` component.

Reply:
That is something that I thought about but ended up not adding. I will add it now!